plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -42,6 +42,12 @@
 {
     public static final String MINIO_ACCESS_KEY = "accesskey";
     public static final String MINIO_SECRET_KEY = "secretkey";
+    /**
+     * In S3 this region is implicitly the default one. In Minio, however,
+     * if we set an empty region, it will accept any.
+     * So setting it by default to `us-east-1` simulates S3 better
+     */
+    public static final String MINIO_DEFAULT_REGION = "us-east-1";
```

Comment:
Is it possible to change it to a non default region ? With the default region set to `us-east-1`. Will the test pass if `hive.s3.region` is not configured ?

Reply:
If we change it here all tests will fail. Unless we change it to empty.
`hive.s3.region` being null will result in `us-east-1` being implicitly selected